### ISSUE TITLE
Build failure: pipecontrol 0.2.8

### ISSUE THREAD

**Issue by toastal** [NixOS/nixpkgs#225978]
### Steps To Reproduce

1. build `pipecontrol` on unstable

### Build log

```
error: build of '/nix/store/icanps6xi7jkp611n9bpplyg7jcw6617-pipecontrol-0.2.8.drv' on 'ssh://olisbokollix-builder' failed: builder for '/nix/store/icanps6xi7jkp611n9bpplyg7jcw6617-pipecontrol-0.2.8.drv' failed with exit code 2;
       last 10 log lines:
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:278: CMakeFiles/pipecontrol.dir/src/pw/qpipewireprofiler.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:166: CMakeFiles/pipecontrol.dir/src/pw/qpipewiremetadata.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:292: CMakeFiles/pipecontrol.dir/src/pw/qpipewirenodelistmodel.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:152: CMakeFiles/pipecontrol.dir/src/pw/qpipewire.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:208: CMakeFiles/pipecontrol.dir/src/pw/qpipewirenode.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:236: CMakeFiles/pipecontrol.dir/src/pw/qpipewiredevice.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:124: CMakeFiles/pipecontrol.dir/pipecontrol_autogen/mocs_compilation.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:138: CMakeFiles/pipecontrol.dir/src/main.cpp.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:138: CMakeFiles/pipecontrol.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/icanps6xi7jkp611n9bpplyg7jcw6617-pipecontrol-0.2.8.drv'.
error: builder for '/nix/store/icanps6xi7jkp611n9bpplyg7jcw6617-pipecontrol-0.2.8.drv' failed with exit code 1;
       last 10 log lines:
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:278: CMakeFiles/pipecontrol.dir/src/pw/qpipewireprofiler.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:166: CMakeFiles/pipecontrol.dir/src/pw/qpipewiremetadata.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:292: CMakeFiles/pipecontrol.dir/src/pw/qpipewirenodelistmodel.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:152: CMakeFiles/pipecontrol.dir/src/pw/qpipewire.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:208: CMakeFiles/pipecontrol.dir/src/pw/qpipewirenode.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:236: CMakeFiles/pipecontrol.dir/src/pw/qpipewiredevice.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:124: CMakeFiles/pipecontrol.dir/pipecontrol_autogen/mocs_compilation.cpp.o] Error 1
       > make[2]: *** [CMakeFiles/pipecontrol.dir/build.make:138: CMakeFiles/pipecontrol.dir/src/main.cpp.o] Error 1
       > make[1]: *** [CMakeFiles/Makefile2:138: CMakeFiles/pipecontrol.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
```

Full log: https://paste.sr.ht/~toastal/311e75ef7b3873e3f33023d42253b5545dd13c1c

### Notify maintainers

@TilCreator 

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.23, NixOS, 23.05 (Stoat), 23.05.20230409.fd531de`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.14.1`
 - channels(toastal): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by MaxHearnden** [comment]
Fixed by 69443ec28a7027d3c73ac471c32bf5875067fca6

**Comment by t4ccer** [comment]
Builds on master (c7c5aaef203550efc74826d41a9dfcdeb7a40e45)
```
nix-build -A pipecontrol
/nix/store/fi923p01b1my37k91vcmvidp1li94l7g-pipecontrol-0.2.11
```
